index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./css/styles.css">
+  <link rel="stylesheet" type="text/css" href="./css/projects.css">
+  <link rel="stylesheet" type="text/css" href="./css/about.css">
+  <link rel="stylesheet" type="text/css" href="./css/form.css">
+  <title>Personal-Profolio</title>
+
+</head>
+
+<body>
+  <!-- Headline Section Starts Here -->
+
+  <header class="navigation">
+    <div class="logo">
+      <h2>My logo</h2>
+    </div>
+    <nav class="navlinks">
+      <img src="/Img/Hlogo.png">
+    </nav>
```

Comment:
Kindly fix the issues with the images that are not showing 🛠, I notice that was because of the `/` at the beginning of the path i.e `"/Img/Hlogo.png"` Kindly remove the `/`. 👍

Your Design
![image](https://user-images.githubusercontent.com/78251927/235106287-86be4452-c388-4217-ab91-8c4cf1434b03.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/235106384-0932be72-f61e-4422-b9fa-645d22d6bebb.png)